app/helpers/application_helper/button/instance_attach.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::InstanceAttach < ApplicationHelper::Button::Basic
+  def disabled?
+    if @record.cloud_tenant.cloud_volumes.where(:status => 'available').count.zero?
+      @error_message = _("There are no %{volumes} available to attach to this %{model}.") % {
```

Comment:
The error message here should be just plain:

`"There are no Cloud Volumes available to attach to this Instance"`

to make life of translators easier.